File: ex2.py
number = int(input("Enter a number: "))

for i in range(number):
    if i == 0:
        print(i + 1)
    else:
        str_val = "1 "
        initial_val = 1
        
        for j in range(i):
            initial_val += 2
            str_val += str(initial_val) + " "
        
        print(str_val)

File: ex2.js
let number = parseInt(prompt("Enter the number of rows: "));
for (let i = 0; i <= number-1; i++) {

if(i == 0) {
    console.log(i+1)
}else {
     let str = 1 + ' ' ;
    let initialVal = 1
    
    for(let j = 0 ; j < i ; j++) {
        initialVal += 2
        str += initialVal + " "
    }
    console.log(str)
}
  
}

